github-for-jira.sd.yml

```diff
@@ -32,12 +32,13 @@ links:
 cleanup: true
 
 resources:
-  - type: redis
-    name: bottleneck
```

Comment:
finally getting rid of this confusing name! :)

Reply:
yarp.